plugins/richcombo/plugin.js

```diff
@@ -74,7 +74,8 @@ CKEDITOR.plugins.add( 'richcombo', {
 				canGroup: false,
 				title: definition.label,
 				modes: { wysiwyg: 1 },
-				editorFocus: 1
+				editorFocus: 1,
+				clearOnBlur: true
```

Comment:
Blur is not a precise word here. I'm keen more towards something like `persistentState` that's more generic.

This is not blur that causes the state check, see https://github.com/jsonn/ckeditor-dev/blob/96f7ebeb8ddf1674c81642e64712b667c5a4a53b/plugins/richcombo/plugin.js#L182-L184